### ISSUE TITLE
Get a strange result using custom dataset to train

### ISSUE THREAD

**Issue by Devin19910617** [faustomorales/keras-ocr#94]
![1](https://user-images.githubusercontent.com/51109294/85266515-86f51180-b4a6-11ea-8fdb-9e5bb68324e1.jpg)
![2](https://user-images.githubusercontent.com/51109294/85266555-95dbc400-b4a6-11ea-97f3-16af4b68749f.jpg)

I use https://keras-ocr.readthedocs.io/en/latest/examples/fine_tuning_recognizer.html to finetune the recognizer,but finally the prediction result is 19486p (the real content is 1986p as picture show)，the picture of 19486p content is in the training set, I get the picture of 1986p content by drawing tool to mask the 4 content, who knows the reason, I will appreciate you very much.

**Comment by faustomorales** [comment]
It would seem as though the model made an error. Closing as this does not appear to be a bug at the moment. Happy to re-open if information to the contrary surfaces.
